concent_api/core/subtask_helpers.py

```diff
@@ -11,133 +11,175 @@
+        )
+    elif subtask.state == Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER.name:  # pylint: disable=no-member
+
+        update_subtask_state(
+            subtask=subtask,
+            state=Subtask.SubtaskState.FAILED.name,  # pylint: disable=no-member
+        )
+        store_pending_message(
+            response_type=PendingResponse.ResponseType.SubtaskResultsRejected,
+            client_public_key=subtask.provider.public_key_bytes,
+            queue=PendingResponse.Queue.ReceiveOutOfBand,
+            subtask=subtask,
+        )
+        store_pending_message(
+            response_type=PendingResponse.ResponseType.SubtaskResultsRejected,
+            client_public_key=subtask.requestor.public_key_bytes,
+            queue=PendingResponse.Queue.ReceiveOutOfBand,
+            subtask=subtask,
+        )
+    elif subtask.state == Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name:  # pylint: disable=no-member
+        task_to_compute = deserialize_message(subtask.task_to_compute.data.tobytes())
+
+        # Worker makes a payment from requestor's deposit just like in the forced acceptance use case.
+        payments_service.make_force_payment_to_provider(  # pylint: disable=no-value-for-parameter
+            requestor_eth_address=task_to_compute.requestor_ethereum_address,
+            provider_eth_address=task_to_compute.provider_ethereum_address,
+            value=task_to_compute.price,
+            payment_ts=get_current_utc_timestamp(),
+        )
+
+        update_subtask_state(
+            subtask=subtask,
+            state=Subtask.SubtaskState.ACCEPTED.name,  # pylint: disable=no-member
+        )
+        store_pending_message(
+            response_type=PendingResponse.ResponseType.SubtaskResultsSettled,
+            client_public_key=subtask.provider.public_key_bytes,
+            queue=PendingResponse.Queue.ReceiveOutOfBand,
+            subtask=subtask,
+        )
+        store_pending_message(
+            response_type=PendingResponse.ResponseType.SubtaskResultsSettled,
+            client_public_key=subtask.requestor.public_key_bytes,
+            queue=PendingResponse.Queue.ReceiveOutOfBand,
+            subtask=subtask,
+        )
+    log_string_message(
+        logger,
+        f"Subtask with id: {subtask.subtask_id} changed it's state from: {subtasks_initial_state} to: {subtask.state}. "
+        f"Provider id: {subtask.provider_id}. Requestor id: {subtask.requestor_id}."
+    )
+
+
+def update_timed_out_subtasks_in_message(client_message: Message, client_public_key: bytes) -> None:
+    if isinstance(client_message, ForcePayment):
+        for subtask_result_accepted in client_message.subtask_results_accepted_list:
+            _update_timed_out_subtask(subtask_result_accepted.subtask_id, client_public_key)
+    else:
+        _update_timed_out_subtask(client_message.subtask_id, client_public_key)
+
+
+def update_all_clients_timed_out_subtasks(client_public_key: bytes) -> None:
```

Comment:
wrong function name: you update just 1 cleint's subtasks

Reply:
done

Reply:
It is still there, why changing the function name in the first place?

Reply:
it was fixed, but not pushed :)